README.md

```diff
@@ -40,3 +40,11 @@ Start Express Server
 ```sh
 npm start
 ```
+
+## CRUD Operations
+| HTTP Verb |           Endpoint          |            Action            |
+|-----------| --------------------------- | ---------------------------- |
+| **POST**  |   /post                     |  CREATE a new item into DB   |
+| **GET**   |   /carousel/:gameId         |  READ data and return data   |
+| **PUT**   |   /put                      |  UPDATE item                 |
+| **DELETE**|   /delete                   |  DELETE item based on ID     |
```

Comment:
I would recommend passing a gameId in to your put/delete requests like you have on the get request. I see you are handling those using faker data, but I don't see a way for a user to use the put/delete endpoints to update/delete a specific record of their own choosing.

Reply:
Thank you for pointing this out. I will fix all of this.